sign_in_token.go

```diff
@@ -0,0 +1,13 @@
+package clerk
+
+type SignInToken struct {
+	APIResource
+	Object    string `json:"object"`
+	ID        string `json:"id"`
+	Status    string `json:"status"`
+	UserID    string `json:"user_id"`
+	Token     string `json:"token,omitempty"`
+	URL       string `json:"url,omitempty"`
```

Comment:
I think the URL can also be `null`.

```suggestion
	URL       *string `json:"url,omitempty"`
```